### ISSUE TITLE
Extra Options for the Calendar

### ISSUE THREAD

**Issue by jasonjac2** [stevencohn/OneMore#1045]
# Problem to Solve

I love the favourites. I love the Navigator (although I am not convinced it doesn't have some issues when left open or on large pages). I use the calendar to to review what I and staff have been updating and what I was doing on certain days.

It is really a list of lists and so I would like to be able to - in order of prefernce:
- Zoom in on day
- Copy an entry to clipboard without having to visit the page.
- Copy all the entries from a day to the clipboard to be able to paste in various formats: into a ON page, word doc , notepad.
- Be able to select all / as many as I want from the display and copy to clipboard.
- Convert the display into a ON page

All of this is for me to be able to review pages and decide if there is anything else required on them.

Niggle: on the list view, the last line bleeds into the next day:
![image](https://github.com/stevencohn/OneMore/assets/68282090/6922dfa5-66eb-4dad-9942-22b9b0743761)

THE FINE PRINT This is a hobby project that I started for myself. I'm happy to take requests and will promise to
consider each one carefully but offer no guarantees that I will ultimately agree to their
usefulness or have time to implement any of them. If OneNote offers a "close enough" work-around
then I will probably reject the request - you've been warned!

**Comment by stevencohn** [comment]
Should Day view be any different than the list view, show more/different fields?
By "copy" do you mean copy links to each page, similar to an index or TOC?

**Comment by jasonjac2** [comment]
Hi Steve, hope things are well with you, when are you looking at for the next release?

> Should Day view be any different than the list view, show more/different fields? 
I think it has everything I need.  It would be better on all of them if I could select text.

>By "copy" do you mean copy links to each page, similar to an index or TOC?
Yes.  But ideally options for:
- Like "Copy link to page" that gives you the same kind of link as "Notebook -> Section -> Page"
- Llain text
- Like a ToC

The whole thing isn't a "must", I just find myself looking at lists and wishing I could interact with them more richely than clicking through them.  The same thing on the "Find / Move/Copy" would be great.

**Comment by jasonjac2** [comment]
I just see all the dialogs that have list on them be better with incremental filters / finds / copy etc.  I was just looking for all the days someone visited "Support TOC xyz" and it was do-able visually, but with the extra functions it would become really easy.

In my line of business application we have a kind of generic part of the context menu and one of the items I intend to implement is "tear off and display in a slice and dice gird" for any list control. It will have incremental filter, formatting options, copy to clipboard options.  Then anything that can generate a multi column list with hyperlinks can feed that "tear off tab".  Sorry, once I start thinking of creative ways of manipulating data, I get a bit carried away. Jason.

**Comment by jasonjac2** [comment]
.... nudge to get some more love on Calander.

**Comment by stevencohn** [comment]
Adding a button to the header of each day in the monthly view that copies links to the pages for that day. It's a start.

<img width="328" alt="image" src="https://github.com/stevencohn/OneMore/assets/9964757/91389c60-5309-4136-94c8-d8503f4974d6">

**Comment by jasonjac2** [comment]
This is a cool feature - thank you. (the copy) - really useful..
It gave me 2 entry in the cal log file.
2nd time I invoked it (with the log open in vs code) I got an exception - Which I can't copy to here, but will get it and add it). It was complaining about clipboard access and the log file had about 1K near identical lines in it.  Shall I open a new ticket?

On this one, the drawing on the month cal is great - a little "bleed" I guess where the scroll bar is/ isn't again I can raise a ticket if you want.

Finally - the list version would be Reall great if it has the same feature against each day, i.e. to copy all from that day (or checkboxes and a copy links button).  This is the view I gravitate to.

Thanks again. 
Aside: I would attach the logs and screens shots, but I am doing this post on a didfferent PC).
